Sources/Apollo/InterceptorProvider.swift

```diff
@@ -16,15 +16,6 @@ public protocol InterceptorProvider {
   func additionalErrorInterceptor<Operation: GraphQLOperation>(for operation: Operation) -> ApolloErrorInterceptor?
 }
 
-/// MARK: - Default Implementation
-
-public extension InterceptorProvider {
-  
-  func additionalErrorInterceptor<Operation: GraphQLOperation>(for operation: Operation) -> ApolloErrorInterceptor? {
-    return nil
-  }
-}
```

Comment:
I think you can add the open funcs to the default interceptor providers below without deleting this default implementation - that way if someone makes their own interceptor provider they don't have to implement this, but if they want to override they can since it's implemented in those implementations.